hadoop-tools/hadoop-aws/src/test/resources/core-site.xml

```diff
@@ -38,6 +38,11 @@
     <description>The endpoint for s3a://landsat-pds URLs</description>
   </property>
 
+  <property>
```

Comment:
can you change the endpoint property above to match?

Reply:
Done.